tests/k8s-azure/manifest/linux-autoscaler.json

```diff
@@ -7,7 +7,8 @@
             "orchestratorRelease": "1.14",
             "kubernetesConfig": {
                 "useCloudControllerManager": true,
-                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0-alpha.1",
+                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0-rc.1",
+                "customHyperkubeImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0-rc.1",
```

Comment:
This should be `gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.14.0-rc.1`

Reply:
fixed.